Dfe.Academies.External.Web/Pages/Trust/FormAMat/ApplicationNewTrustPlansForGrowth.cshtml.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Dfe.Academies.External.Web.Enums;
+using Dfe.Academies.External.Web.Models;
+using Dfe.Academies.External.Web.Pages.Base;
+using Dfe.Academies.External.Web.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Dfe.Academies.External.Web.Pages.Trust.FormAMat
+{
+	public class ApplicationNewTrustPlansForGrowthModel : BaseTrustFamApplicationPageEditModel
+	{
+		public string TrustName { get; private set; } = string.Empty;
+
+		// MR:- VM props to capture data
+
+		[BindProperty]
+		[Required(ErrorMessage = "You must select an option")]
+		public SelectOption GrowthPlansOption { get; set; }
+
+		[BindProperty]
+		public string GrowthPlanDescription { get; set; } = string.Empty;
+
+		[BindProperty]
+		public string NoGrowthPlanDescription { get; set; } = string.Empty;
```

Comment:
I think this might need to be string? otherwise built in MVC validation always expects it?

Reply:
done